src/components/Item/AppItem/getIframeSrc.js

```diff
@@ -11,5 +11,12 @@ export const getIframeSrc = (appDetails, item, itemFilters) => {
         iframeSrc += `&userOrgUnit=${ouIds.join(',')}`
     }
 
+    // Add period (pe) to iframeSrc
+    if (itemFilters[FILTER_PE] && itemFilters[FILTER_PE].length) {
+        const peValues = itemFilters[FILTER_PE].map(x => x.id).join(';');
+        iframeSrc += `&pe=${peValues}`;
```

Comment:
To match the org unit parameter, let's spell out `period`:
```suggestion
        iframeSrc += `&period=${peValues}`;
```